packages/sources/iex-cloud-test/src/endpoint/crypto.ts

```diff
@@ -0,0 +1,82 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { PriceEndpoint, PriceEndpointParams } from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { customSettings } from '../config'
+
+interface ResponseSchema {
+  symbol: string
+  primaryExchange: string
+  sector: string
+  calculationPrice: string
+  high: string
+  low: string
+  latestPrice: string
+  latestSource: string
+  latestUpdate: number
+  latestVolume: string
+  previousClose: string
+}
+
+export type CryptoEndpointTypes = {
+  Request: {
+    Params: PriceEndpointParams
+  }
+  Response: SingleNumberResultResponse
+  CustomSettings: typeof customSettings
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin', 'asset', 'symbol'],
+    description: 'The symbol of symbols of the currency to query',
+    required: true,
+    type: 'string',
+  },
+  quote: {
+    aliases: ['to', 'market'],
+    description: 'The symbol of the currency to convert to',
+    required: true,
+    type: 'string',
+  },
+} as const
+
+export const httpTransport = new HttpTransport<CryptoEndpointTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      return {
+        params,
+        request: {
+          url: `/crypto/${param.base.toUpperCase()}${param.quote.toUpperCase()}/quote`,
+          baseURL: config.API_ENDPOINT,
+          params: {
+            token: config.API_KEY,
+          },
+        },
+      }
+    })
+  },
+  parseResponse: (params, res) => {
+    return params.map((param) => {
+      const result = Number(res.data.latestPrice)
+      return {
+        params: param,
+        response: {
+          data: {
+            result: result,
+          },
+          result,
+        },
+      }
```

Comment:
We should maybe do a validation on latestPrice like if it exists or not and return an error